src/dotnet/commands/dotnet-update/tool/LocalizableStrings.resx

```diff
@@ -0,0 +1,183 @@
+  </resheader>
+  <data name="PackageIdArgumentName" xml:space="preserve">
+    <value>PACKAGE_ID</value>
+  </data>
+  <data name="PackageIdArgumentDescription" xml:space="preserve">
+    <value>NuGet Package Id of the tool to update.</value>
+  </data>
+  <data name="SpecifyExactlyOnePackageId" xml:space="preserve">
+    <value>Please specify one tool Package Id to update.</value>
+  </data>
+  <data name="VersionOptionDescription" xml:space="preserve">
+    <value>Version of the tool package in NuGet.</value>
+  </data>
+  <data name="SourceOptionDescription" xml:space="preserve">
+    <value>Specifies a NuGet package source to use during update.</value>
+  </data>
+  <data name="SourceOptionName" xml:space="preserve">
+    <value>SOURCE</value>
+  </data>
+  <data name="CommandDescription" xml:space="preserve">
+    <value>Updates a tool for use on the command line.</value>
+  </data>
+  <data name="ConfigFileOptionDescription" xml:space="preserve">
+    <value>The NuGet configuration file to use.</value>
+  </data>
+  <data name="FrameworkOptionDescription" xml:space="preserve">
+    <value>The target framework to update the tool for.</value>
+  </data>
+  <data name="NuGetConfigurationFileDoesNotExist" xml:space="preserve">
+    <value>NuGet configuration file '{0}' does not exist.</value>
+  </data>
+  <data name="UpdateSucceeded" xml:space="preserve">
+    <value>Tool '{0}' (from version '{1}' to version'{2}') was successfully updated.</value>
+  </data>
+  <data name="GlobalOptionDescription" xml:space="preserve">
+    <value>Update user wide.</value>
+  </data>
+  <data name="UpdateFullCommandNameLocalized" xml:space="preserve">
+    <value>.NET update Command</value>
+  </data>
+  <data name="ToolAlreadyInstalled" xml:space="preserve">
+    <value>Tool '{0}' is already installed.</value>
+  </data>
+  <data name="UpdateToolCommandNeedGlobalOrToolPath" xml:space="preserve">
+    <value>Please specify either the global option (--global) or the tool path option (--tool-path).</value>
+  </data>
+  <data name="UpdateToolCommandInvalidGlobalAndToolPath" xml:space="preserve">
+    <value>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</value>
+  </data>
+  <data name="ToolPathDescription" xml:space="preserve">
+    <value>Location of shim to access tool</value>
+  </data>
+  <data name="ToolHasMultipleVersionsInstalled" xml:space="preserve">
+    <value>Tool '{0}' has multiple versions installed and cannot be uninstalled.</value>
+  </data>
+  <data name="ToolNotInstalled" xml:space="preserve">
+    <value>Tool '{0}' is not currently installed.</value>
+  </data>
+  <data name="UpdateSucceededVersionNoChange" xml:space="preserve">
+    <value>Tool '{0}' was successfully updated with no version change.</value>
+  </data>
+  <data name="UpdateToolFailed" xml:space="preserve">
```

Comment:
Perhaps `Tool '{0}' failed to update due to the following errors:` (this should be a single sentence that ends with a colon).

Reply:
Agree

Reply:
Changed as suggested